tests/settings.spec.ts

```diff
@@ -15,29 +15,30 @@ const SETTINGS_MENU = {
   ROUTE_PRIORITY: {
```

Comment:
Same comment as before regarding this, do not change files that you are not updating please